### PR TITLE
[Doc] correct ceph adoption prerequisites

### PR DESCRIPTION
**Pull request by katarimanojk**
update ceph rgw, rbd and nfs prerequisites steps in the adoption document.

Jira: https://issues.redhat.com/browse/OSPRH-11496

### CODE REVIEW

File: docs_user/modules/proc_completing-prerequisites-for-migrating-ceph-rbd.adoc
Comment (fmount) [inline docs_user/modules/proc_completing-prerequisites-for-migrating-ceph-rbd.adoc:19]: Thank you, good to stay consistent w/ the 17.1 documentation. @klgill to review

File: docs_user/modules/proc_completing-prerequisites-for-migrating-ceph-rgw.adoc
Comment (fmount) [inline docs_user/modules/proc_completing-prerequisites-for-migrating-ceph-rgw.adoc:58]: +1 <2> refers to the storage network (which is vlan30). Thanks for fixing this.

File: docs_user/modules/proc_completing-prerequisites-for-migrating-ceph-rgw.adoc
Comment (fmount) [inline docs_user/modules/proc_completing-prerequisites-for-migrating-ceph-rgw.adoc:102]: I'm not sure `Notes` have this syntax: @klgill to check and ack/nack the proposed content.

File: docs_user/modules/proc_completing-prerequisites-for-migrating-ceph-rgw.adoc
Comment (fmount) [inline docs_user/modules/proc_completing-prerequisites-for-migrating-ceph-rgw.adoc:149]: I think we should ask to always pass `network_config_update` to the templates.
Do we want to call this out with a Note instead of adding this statement on L149?
Basically I'm proposing to keep the the title as it was, and put a Note saying "you must add `network_config_update: true` to make sure `os-net-config` is triggered.
@klgill might definitely have a better proposal.

File: docs_user/modules/proc_creating-a-ceph-nfs-cluster.adoc
Comment (fmount) [inline docs_user/modules/proc_creating-a-ceph-nfs-cluster.adoc:16]: same as before.
@klgill to review/suggest the right way of passing this note.

File: docs_user/modules/proc_completing-prerequisites-for-migrating-ceph-rbd.adoc
Comment (klgill) [inline docs_user/modules/proc_completing-prerequisites-for-migrating-ceph-rbd.adoc:19]:
```suggestion
bare metal file for the `CephStorage` nodes, for example, `/home/stack/composable_roles/network/baremetal_deployment.yaml`:
```

File: docs_user/modules/proc_completing-prerequisites-for-migrating-ceph-rgw.adoc
Comment (klgill) [inline docs_user/modules/proc_completing-prerequisites-for-migrating-ceph-rgw.adoc:149]:
```suggestion
.. Add the External Network to the bare metal file, for example, `/home/stack/composable_roles/network/baremetal_deployment.yaml` that is used by `metalsmith`:
+
[NOTE]
Ensure that  'network_config_update' is enabled for network propagation to the target nodes during `os-net-config`.
```
What do you mean by *during* `os-net-config`? Do we want to say something like "... is enabled for network propagation to the target nodes when `os-net-config` is triggered" ?

File: docs_user/modules/proc_creating-a-ceph-nfs-cluster.adoc
Comment (klgill) [inline docs_user/modules/proc_creating-a-ceph-nfs-cluster.adoc:16]:
```suggestion
+
[NOTE]
If the target nodes are not managed by director, you cannot use this procedure to configure the network. An administrator must manually configure all the required networks.
```

**Review by klgill — CHANGES_REQUESTED** [review]
I made a few edits.

**Comment by klgill** [comment]
Note to self: After this PR is merged, I need to copy the updated files downstream and republish the adoption guide.

File: docs_user/modules/proc_completing-prerequisites-for-migrating-ceph-rgw.adoc
Comment (katarimanojk) [reply docs_user/modules/proc_completing-prerequisites-for-migrating-ceph-rgw.adoc:149]: >>What do you mean by during os-net-config? Do we want to say something like "... is enabled for network propagation >>to the target nodes when os-net-config is triggered" ?

@klgill yes you are right.

**Review by jistr — APPROVED** [review]
Holding off merge for addressing reviews but overall LGTM.

File: docs_user/modules/proc_completing-prerequisites-for-migrating-ceph-rbd.adoc
Comment (fmount) [reply docs_user/modules/proc_completing-prerequisites-for-migrating-ceph-rbd.adoc:19]: @katarimanojk FYI ^

File: docs_user/modules/proc_completing-prerequisites-for-migrating-ceph-rgw.adoc
Comment (fmount) [reply docs_user/modules/proc_completing-prerequisites-for-migrating-ceph-rgw.adoc:102]: @katarimanojk FYI ^

File: docs_user/modules/proc_creating-a-ceph-nfs-cluster.adoc
Comment (fmount) [reply docs_user/modules/proc_creating-a-ceph-nfs-cluster.adoc:16]: @katarimanojk FYI ^

File: docs_user/modules/proc_completing-prerequisites-for-migrating-ceph-rgw.adoc
Comment (katarimanojk) [reply docs_user/modules/proc_completing-prerequisites-for-migrating-ceph-rgw.adoc:108]: done

File: docs_user/modules/proc_completing-prerequisites-for-migrating-ceph-rgw.adoc
Comment (klgill) [inline docs_user/modules/proc_completing-prerequisites-for-migrating-ceph-rgw.adoc:154]:
```suggestion
Ensure that 'network_config_update' is enabled for network propagation to the target nodes when `os-net-config` is triggered.
```

**Review by klgill — CHANGES_REQUESTED** [review]
1 minor comment.

File: docs_user/modules/proc_completing-prerequisites-for-migrating-ceph-rgw.adoc
Comment (katarimanojk) [reply docs_user/modules/proc_completing-prerequisites-for-migrating-ceph-rgw.adoc:154]: done

**Review by fmount — APPROVED** [review]
/lgtm

**Pull request merged**
